src/components/display/DisplayImage.js

```diff
@@ -2,60 +2,63 @@ import axiosInstance from "../api/axios";
 import { useState, useEffect } from "react";
 import "./DisplayImage.css";
 
-const DisplayImage = ({ images, setImages }) => {
- 
-  // const [showMore, setShowMore] = useState(5);
+const DisplayImage = ({ images, setImages, err, setErr }) => {
+  const [perPage, setPerPage] = useState("");
 
   useEffect(() => {
     const showImage = async () => {
       try {
-        const res = await axiosInstance.get();
-        setImages(res.data.results);
-        console.log(res);
-      } catch (error) {
-        console.log(error);
+        const { data } = await axiosInstance.get(`?&offset=${perPage}`);
+        console.log(data)
+        setImages(data.results);
+        if(!data.next){
+          setErr("No more images")
+        }
+      } catch {
+        setErr("Error showing images");
       }
     };
     showImage();
-  }, []);
+  }, [setImages, perPage, setErr]);
+
+  const loadMore = () => {
+    setPerPage((more) => more + 1); //not perfect yet
+  };
 
   const handleDelete = async (id) => {
     try {
-      const result = await axiosInstance.delete(id);
+      const result = await axiosInstance.delete(`${id}`);
       console.log(result);
     } catch (error) {
       console.log(error);
     }
   };
 
-  const loadMore = async () => {
-    // try {
-    //   const next = await axiosInstance.get("/")
-    //   setShowMore(next.data.results)
-    //   console.log(setShowMore)
-    // } catch (error) {
-    //   console.log(error)
-    // }
-  };
-
   return (
     <>
       <div className="display-section">
         <div className="image-section">
+          <div className="error-container">
+            <div className="error-display">{err}</div>
+          </div>
```

Comment:
Should this not be conditionally rendered?

Reply:
adjusted